src/App.tsx

```diff
@@ -429,8 +425,6 @@ export default function App() {
       <ToastSnackbar />
       <TopBar>
         <TopBarContent
-          notifications={notifications}
```

Comment:
Oh, were we passing an empty object?

Reply:
we were holding the notifications state in the App (i think because previously the information was shared across components). we don't share this info anymore.